src/docs/publish_via_git.sh

```diff
@@ -32,7 +32,7 @@ path_within_url=$2
 out=/tmp/pantsdoc.$$
 
 # When done, clean up tmp dir:
-trap "rm -fr $out" 0 1 2
+trap 'rm -fr $out' 0 1 2
```

Comment:
This one might be wrong.. Refer to https://github.com/koalaman/shellcheck/wiki/SC2064 and let me know what you think.

Reply:
I think this works! The `trap` quoting rules confuse me greatly in general.

Reply:
AIUI because we don't change the value of `$out`, these are equivalent here.

Reply:
(I didn't know about `trap` quoting rules until last week!)

Reply:
Agreed. Thanks for the check!